jwst/tweakreg/tweakreg_step.py

```diff
@@ -279,35 +279,30 @@ def process(self, input):
             g = grp_img[0]
             if len(g) == 0:
                 raise AssertionError("Logical error in the pipeline code.")
-            group_name = _common_name(g)
             imcats = list(map(self._imodel2wcsim, g))
             # Remove the attached catalogs
             for model in g:
                 del model.catalog
-            self.log.info("* Images in GROUP '{}':".format(group_name))
+            self.log.info(f"* Images in GROUP '{model.meta['group_id']}':")
```

Comment:
```suggestion
            self.log.info(f"* Images in GROUP '{model.meta.group_id}':")
```
I think `model` is a `DataModel` (not a corrector) here so the `group_id` access needs to be through the attribute (not via a subscript).